c/call-bind/call-bind_ubi8.4.sh

```diff
@@ -0,0 +1,51 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : call-bind
+# Version       : master(a994df69f401f4bf735a4ccd77029b85d1549453)
+# Source repo   : https://github.com/ljharb/call-bind.git
+# Tested on     : UBI 8.4
+# Script License: MIT License
```

Comment:
this is script license, not the package license, should be Apache 2.0

Reply:
Updated License